packages/template-retail-react-app/app/hooks/use-navigation.test.js

```diff
@@ -19,6 +19,10 @@ jest.mock('pwa-kit-runtime/utils/ssr-config', () => {
 
 const mockHistoryPush = jest.fn()
 const mockHistoryReplace = jest.fn()
+Object.defineProperty(window, 'location', {
```

Comment:
There's an open bug with jest on Node 18.16 that prevents modification of global objects. The issue has been fixed in Node 19 but is not yet back-ported to Node 18.x
This is a temporary fix to prevent errors like: `Cannot redefine window in global.window`

Reply:
Do you want to add a comment in the code to remind us of this?